help.html

```diff
@@ -220,20 +243,23 @@ <h2><Strong>Details</Strong></h2>
                     </script>
                 </div>
                 <div class="help-right">
-                    <h2><strong>Reach Us</strong></h2>
-                    <br>
+
+                    <h2 style="text-align: center; text-decoration: underline; font-family: 'Baloo Tammudu 2', cursive;">REACH US</h2>
```

Comment:
@skpandey885 you had used inline styling most of the place please convert it to class based styling.